### ISSUE TITLE
the default seurat_clusters isn't consistent with the updated RNA_snn_res(should be default)

### ISSUE THREAD

**Issue by xinmiaoyan** [satijalab/seurat#6257]
I've runned seurat with different resolution to cluster cells, and the result showed inconsistent when plot the dimple with the default seurat_clusters and the same resolution RNA_snn_res. 
the code is :
`
obj = FindNeighbors(obj, dims = pc.num) 
obj = FindClusters(obj, resolution = c(0.1,0.2,0.4,0.6,0.8,1))
obj = RunUMAP(obj, dims = pc.num)`

so the default seurat_clusters should be the same as RNA_snn_res.1, but the dimplot shows a messy clustering with the default seurat_clusters. I've checked the table, it clear shows the issue, appreciate it if you could help.
<img width="430" alt="Screen Shot 2022-08-02 at 12 57 57 PM" src="https://user-images.githubusercontent.com/83053884/182442259-863151ca-c00c-4ce7-8067-aae6bc0a0b94.png">

**Comment by xinmiaoyan** [comment]
please check the plot detail.
plot with group_by = 'RNA_snn_res.1'
![image](https://user-images.githubusercontent.com/83053884/182634386-5b088612-2a1b-499f-b7a5-0cf0d438404a.png)
plot with default parameter
![image](https://user-images.githubusercontent.com/83053884/182634472-d0466f8b-b63d-477a-b78b-be9eb2cdf369.png)
I also tired Idents(obj) = 'RNA_snn_res.1' to correct default clusters, but it doesn't work. 
appreciate it if you could help.

**Comment by xinmiaoyan** [comment]
I checked all the possibilities, and found out the rownames(obj@meta.data) isn't the same order of colnames(obj) since I've added information to meta.data and changed the order of rownames. and after I corrected this order, the problem seems fixed. 
so the lesson is every time when I changed meta.data, double check and make sure the order of  rownames(obj@meta.data) is the same as colnames(obj)
`identical(rownames(obj@meta.data),colnames(obj))`

Thanks, this issue is closed right now.
